test/test_helpers.rb

```diff
@@ -35,6 +35,7 @@ def run(*args, &block)
 
   before do
     Thread.current[:test_run_uid] = @test_run_uid = self.class.to_s.underscore + "/" + name.gsub(/^test_[0-9]+_/, 'test_it_') + "/" + SecureRandom.hex
+    $test_run_uid = SecureRandom.uuid.freeze
```

Comment:
Define this with caching
`$test_run_uid ||= SecureRandom.uuid`
No need to call `freeze` on this